web/bfex/components/key_generation/__init__.py

```diff
@@ -0,0 +1,15 @@
+__all__ = []
+
+import pkgutil
+import inspect
+
+for loader, name, is_pkg in pkgutil.walk_packages(__path__):
+    module = loader.find_module(name).load_module(name)
+
```

Comment:
Where did this code come from? Did you write it yourself? I like it a lot.

Reply:
Yea... I need to get used to that more often